Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py

```diff
@@ -851,11 +855,10 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
         incident_created_time_ms = incident_created_time.timestamp()
 
         # to prevent duplicates, adding incidents with creation_time > last fetched incident
-        if last_fetch:
-            if incident_created_time_ms <= last_fetch.timestamp():
-                demisto.debug(f'{INTEGRATION_NAME} - alert {str(alert)} was created at {incident_created_time_ms}.'
-                              f' Skipping.')
-                continue
+        if last_fetch and incident_created_time_ms <= last_fetch.timestamp():
```

Comment:
```suggestion
        if last_fetch and (incident_created_time_ms <= last_fetch.timestamp()):
```
mostly for readability, and to reduce chance of misinterpretation